pkg/controllers/vdb/init_db.go

```diff
@@ -395,8 +395,7 @@ func (g *GenericDatabaseInitializer) getAdditionalConfigParmsContent(content str
 		// we skip to the next parm.
 		_, ok := parmNames[strings.ToLower(k)]
 		if ok {
-			g.VRec.Eventf(g.Vdb, corev1.EventTypeWarning, events.AdditionalConfigParmIgnored,
```

Comment:
I'm not sure if you were ready for a re-review, but we should remove this event you added.